core-tests/shared/src/test/scala/zio/ZLayerSpec.scala

```diff
@@ -61,14 +61,14 @@ object ZLayerSpec extends ZIOBaseSpec {
   def spec =
     suite("ZLayerSpec")(
       testM("Size of >>> (1)") {
-        val layer = ZLayer.succeed(1) >>> ZLayer.fromService((i: Int) => Has(i.toString))
+        val layer = ZLayer.succeed(1) >>> ZLayer.fromServiceMany((i: Int) => Has(i.toString))
```

Comment:
This one and the other ones can be simplified to `fromService` since they don't produce many.

Reply:
Good catch! Fixed now.